src/main/java/core/basesyntax/App.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class App {
+    final static int FIGURES_LENGTH = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURES_LENGTH];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < figures.length; i++) {
+                figures[i] = figureSupplier.getRandomFigure(i <= 2);
```

Comment:
The condition `i <= 2` is used to decide whether to generate a random figure or a default one. However, this logic is incorrect according to the task requirements. The task specifies that the first half of the figures should have random parameters, and the second half should have default parameters. Consider adjusting the condition to reflect this requirement.